shop/shop-controller/src/main/java/shop/woowasap/shop/controller/CartController.java

```diff
@@ -0,0 +1,24 @@
+package shop.woowasap.shop.controller;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import shop.woowasap.shop.domain.api.cart.CartUseCase;
+import shop.woowasap.shop.domain.api.cart.response.CartResponse;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/v1/carts")
+public class CartController {
+
+    private final static long MOCK_USER_ID = 1L;
+    private final CartUseCase cartUseCase;
```

Comment:
요기 한칸 띄워주는게 컨벤션 인거 같아요 👍

```suggestion
    private final static long MOCK_USER_ID = 1L;
    
    private final CartUseCase cartUseCase;
```